src/app/shared/components/template/components/points-item/points-item.component.scss

```diff
@@ -1,10 +1,10 @@
 .wrapper-point {
```

Comment:
.wrapper-point {
    --scale-factor--point: 1;
}
@media screen and (max-width: 400px) {
  .wrapper-point {
    --scale-factor--point: 0.227vw; // I think this is the right number 100 vw / 440
  }
}

Reply:
But now it is dynamically and calculates based on template-container width. This is more universal solution from my point of view

Reply:
I don't think that's true, but merging anyway